junit/src/main/java/org/kaazing/k3po/junit/rules/Latch.java

```diff
@@ -93,7 +93,7 @@ void notifyFinished() {
         switch (state) {
         case INIT:
             notifyPrepared();
-            break;
+//            break;
```

Comment:
We have a call to notifyFinished, it seems normal to me that at the end of this call the state should be FINISHED.
Anyway, the real problem is that we need a flag to check if the thread reading the events in a loop (the ScriptRunner) is finished. I considered it to be the finished latch. Otherwise we will have to add another one, maybe something like stopped and set it when the notifyFinished is called. From my point of view, I don't see why we cannot use finished instead.